app/Image/SizeVariantDefaultFactory.php

```diff
@@ -277,15 +277,23 @@ protected function createSizeVariantInternal(int $sizeVariant, int $maxWidth, in
 
 		$sv = $this->photo->size_variants->getSizeVariant($sizeVariant);
 		if (!$sv) {
-			$sv = $this->photo->size_variants->create($sizeVariant, $shortPath, $maxWidth, $maxHeight);
+			// Filesize is not known before creating the file, set to 0 meanwhile
+			$sv = $this->photo->size_variants->create($sizeVariant, $shortPath, $maxWidth, $maxHeight, 0);
```

Comment:
by using a default value, you no longer need to add the argument here.

Reply:
Done.